src/workerd/jsg/jsg.h

```diff
@@ -2172,6 +2172,8 @@ class Lock {
 
   BufferSource bytes(kj::Array<kj::byte> data) KJ_WARN_UNUSED_RESULT;
 
+  BufferSource arrayBuffer(kj::Array<kj::byte> data) KJ_WARN_UNUSED_RESULT;
```

Comment:
Nit: I'd suggest perhaps either a comment explaining the difference between `bytes()` and `arrayBuffer()`, or renaming `bytes()` to `uint8Array()` to make it obvious.